Overhaul Site Package tutorial (#143)
TYPO3-Documentation/TYPO3CMS-Tutorial-SitePackage: Documentation/ExtensionInstallation/Index.rst

This tutorial assumes that your TYPO3 instance is a brand new installation,

> without any themes, templates, pages or content. See the :doc:`TYPO3

Author: ```suggestion
without any themes, templates, pages or content. See the :doc:`Getting Started Tutorial <t3start:Index>`
```

> Installation Guide <t3install:Index>` for a detailed explanation on how to set up

Author: ```suggestion
for a detailed explanation on how to set up
```

a TYPO3 instance from scratch.

We highly recommend using :ref:`the Composer -based installation process
<t3start:install-extension-with-composer>`. During development you should work locally

> on your machine, e.g. by running TYPO3 on ddev.

Reviewer: We want to omit "e.g." and use "for example".

> on your machine, e.g. by running TYPO3 on ddev.

Author: ```suggestion
on your machine, for example by running TYPO3 on ddev.
```

Extension installation in Composer mode
=======================================

> From TYPO3 11.4 if composer is used, all extensions, including our

Reviewer: ```suggestion
From TYPO3 v11.4 if composer is used, all extensions, including our
```

I had to read the sentence twice. Maybe we can rephrase it, like "In Composer-based installations, all extensions ...".

site package extension, must be installed via Composer.

As a site package is created with site-specific files it is usually best to keep
the files together in a version control system such as Git.

Create a directory for local version-controlled extensions at the
root-level of your Composer-based installation. The name is arbitrary, we use

> :file:`packages` here.

Reviewer: ```suggestion
:file:`packages/` here.
```

We end folders with a "/".

Extension installation in Composer mode
=======================================

> From TYPO3 v11.4 if composer is used, all extensions, including our

Reviewer: ```suggestion
From TYPO3 v11.4 if Composer is used, all extensions, including our
```

      "..." : "..."
   }

> Move your extension folder :file:`site_package` into the :file:`packages`

Reviewer: ```suggestion
Move your extension folder :file:`site_package/` into the :file:`packages/`
```

folder. Then *require* the extension via Composer using the
package name defined in the site package extension's :file:`composer.json` now located
at :file:`packages/site_package/`

.. code-block:: json
   :caption: packages/site_package/composer.json

   {
      "name": "myvendor/site-package"
   }

require it by:

.. code-block:: bash
   :caption: Execute in directory page_root

    composer require myvendor/site-package:@dev

.. hint::

>    From TYPO3 11.4 all extensions required via Composer are

Reviewer: ```suggestion
   From TYPO3 v11.4 all extensions required via Composer are
```
